### ISSUE TITLE
[UX] Unclickable dashboard plates + misleading "share to" icon

### ISSUE THREAD

**Issue by fadeevab** [NEAR-DevHub/neardevhub-bos#196]
I was surfing through the Near Social boards and I was attempting to click on the items, however, the items appear to be **unclickable**:
I was suggested to click on the button which means **"share to" on the major of other social platforms**:

![image](https://github.com/near/neardevhub-widgets/assets/5967447/9e08c05f-065d-4bbf-9224-341ad0ccda45)

⚠I claim that this UX is **counter-intuitive and consumes time.**

**Describe the solution you'd like**
☝ Make the item plate clickable: **clicking on the white space and/or a title should open the item.**

**Describe alternatives you've considered**
😔 Change the icon.

**Comment by jaswinder6991** [comment]
I agree with the OP that the icon for "Open in new tab" seems like share button as that's how it is used on other platforms. We need to use something like the bootstrap's glyph icon. See this codepen [example](https://codepen.io/marklchaves/pen/MWWoVGG).  
I will update the issue with the design after talking to Sasha who worked on the designs for the new DevHub app.
